src/lib/components/modals/SubmitBugModal.svelte

```diff
@@ -0,0 +1,18 @@
+<script lang="ts">
+  import SupportModal from "./SupportModal.svelte";
+
+  export let isVisible = '';
+
+</script>
+
+<SupportModal
+  bind:isVisible={isVisible}
+  description='Please describe the bug'
+>
+  <svelte:fragment slot="description">
+    <p>
+      Oops you’ve found a bug!
+    </p>
+    <p>You can report the issue and our team will get it resolved.</p>
```

Comment:
I would modify this slightly to:

"You can report the issue here and our team will get it resolved right away."